auditbeat/Jenkinsfile.yml

```diff
@@ -60,6 +60,11 @@ stages:
 #                 - "macosM1Test"
 #             tags: false         ## for all the tags
 #         stage: extended
+    rhel-9:
```

Comment:
I suggest adding this to x-pack/auditbeat/Jenkinsfile.yml as well so that we are testing the system module on RHEL 9 too.

Reply:
Okay. In `x-pack/packetbeat/Jenkinsfile.yml` as well? I did that and updated the branch.

Reply:
Okay. In `x-pack/packetbeat/Jenkinsfile.yml` as well? I did that and updated the branch.